lib/smack/IntegerOverflowChecker.cpp

```diff
@@ -124,7 +124,8 @@ bool IntegerOverflowChecker::runOnModule(Module& m) {
         if (auto chkshft = dyn_cast<CallInst>(&*I)) {
 	  Function* chkfn = chkshft->getCalledFunction();
           if (chkfn && chkfn->hasName() &&
-              chkfn->getName().find("__ubsan_handle_shift_out_of_bounds") != std::string::npos) {
+              (chkfn->getName().find("__ubsan_handle_shift_out_of_bounds") != std::string::npos ||
+               chkfn->getName().find("__ubsan_handle_divrem_overflow") != std::string::npos)) {
             // If the call to __ubsan_handle_shift_out_of_bounds is reachable,
```

Comment:
In the should just say these functions are only called by ubsan when an error occurs.